editor/backend/package.json

```diff
@@ -70,7 +68,9 @@
     "ts-node": "^7.0.1",
     "typescript": "~3.7.4"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@bentley/imodeljs-editor-common": "2.11.0-dev.20"
```

Comment:
This should probably stay as a peerDep otherwise you could get multiple versions when pulling in multiple commands that would break the types.

Reply:
I think this should be a regular dependency (it is the "host") and all command implementations should use peer dependencies. Do I misunderstand?

Reply:
@calebmshafer Brien needs this PR. I'll change this in another one if this turns out to be wrong.

Reply:
Okay.  If I'm understanding this correctly, each command package will need to have a peerDep on the `imodeljs-editor-backend` because it has to derive from `EditCommand` and the app will drive that version so there aren't multiple versions.

If so, we'd want to follow the same pattern with the `imodeljs-editor-common` since in the pattern described above the common version could potentially differ between editor-backend dependencies depending on how npm handles it.